Makefile

```diff
@@ -10,6 +10,7 @@ installProd:
 	npm ci --only=production
 
 developmentTests:
+	npm run audit:ci
```

Comment:
Can this please be run after the other tests? It's a better dev experience if npm audit becomes flaky again and times out, to have the unit tests and storybook e2e tests passing first.

Reply:
good idea.